components/date-picker/RangePicker.tsx

```diff
@@ -223,6 +224,7 @@ class RangePicker extends React.Component<any, RangePickerState> {
       return (
         <Tag
           key={range}
+          prefixCls={tagPrefixCls}
```

Comment:
You can just pass the `prefixCls` of RangePicker to Tag.

Reply:
The prefixCls of RangePicker default value is "ant-calendar".
I wish could custom the tag prefuxCls of Tag.

Reply:
Please read the [comment](https://github.com/ant-design/ant-design/commit/00afce3a4bf3085fe8309435a03d2597a519bef8#diff-0f7c946177eb7f19b2586e4cb67ee699), thanks.

Reply:
👌, new feature should be sent to `feature-3.8.0`.

Reply:
Sorry, this is my first commit, I will try commit to feature-3.8.0, thanks.